packages/web/src/pages/upload-page/fields/ReleaseDateField.module.css

```diff
@@ -60,7 +60,7 @@
   background-color: var(--neutral-light-10);
   border: 1px solid var(--neutral-light-9);
   border-radius: 4px;
-  width: 100%;
+  width: 200%;
```

Comment:
is this intended? `200%` sort of unusual

Reply:
yea just realized the date picker should be twice as wide but maybe i should set it to fixed pixels

Reply:
likely the right approach is to make the parent wider and keep this at 100%